src/main/java/com/js/tictactoe/game/Game.java

```diff
@@ -83,50 +82,54 @@ private void switchPlayers() {
   }
 
   private boolean move() {
-    boolean isWinner;
+    boolean isWinner = false;
     int i = 0;
     do {
-      board.printBoard(output);
-      output.accept(reader.loadString("player") + currentPlayer.getName() + "(" + currentPlayer.getSign() + ")" +
-              reader.loadString("move") + board.getWidth() + reader.loadString("movePt2") + board.getHeight() + ": ");
-      boolean added;
-      do {
+      printOutputDuringMove();
+      String line = DigitParser.correctCoordinates(input, output, reader);
 
-        String line = DigitParser.correctCoordinates(input, output, reader);
+      if(isExit(line)) {
+        match.endMatch();
+        return false;
+      }
 
-        if (isExit(line)) {
-          return false;
-        }
+      boolean isAdded = tryToMakeMove(line);
+      if (isAdded) {
+        isWinner = judge.isWinner(currentPlayer.getSign());
+        i++;
+        switchPlayers();
+      }
 
-        added = makeMove(line, currentPlayer);
+    } while (!isWinner && i < board.getHeight() * board.getWidth() && match.isNextRound());
```

Comment:
board.getHeight() * board.getWidth() - extract to variable

Reply:
done